babel.config.js

```diff
@@ -1,8 +1,32 @@
-module.exports = {
-  "env": {
-    "test": {
-      "presets": ["@babel/preset-env", "@babel/preset-react"],
-      "plugins": ["@babel/plugin-transform-runtime"]
-    }
-  }
-}
+module.exports = function (api) {
+  api.cache(true);
+
+  const presets = [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          browsers: ['Chrome >= 60', 'Firefox >= 44', 'Safari >= 8', 'Edge >= 17'],
```

Comment:
In addition to Safari, it could be worth to look at the minimal version of Firefox we need to support.  

With
```diff
-          browsers: ['Chrome >= 60', 'Firefox >= 44', 'Safari >= 8', 'Edge >= 17'],
+          browsers: ['Chrome >= 60', 'Firefox >= 60', 'Safari >= 10', 'Edge >= 17'],
```
the size of the first bundle would shrink from 527KB to 420KB.